docs/DeveloperGuide.md

```diff
@@ -1479,10 +1491,112 @@ testers are expected to do more *exploratory* testing.
+   1. Test case: With all fields supplied e.g. `edit ct/w i/1 n/Alice Warehouse p/82345162 addr/21 Lower Kent Ridge Rd, Singapore 119077 r/Largest Warehouse`<br>
+      Expected: Edits a warehouse in index 1 on warehouse list with all the fields applied.
+   1. Test case: Invalid Prefix or missing compulsory Prefixes e.g. `edit ct/w i/1 n/Alice Warehouse p/82345162 addr/21 Lower Kent Ridge Rd, Singapore 119077 z/large `
+      or `edit ct/w i/1`<br>
+      Expected: No warehouse is added. Error details shown in the response message. A help message for edit command will also be displayed
+      to guide user accordingly. WarehouseList on GUI remains unchanged.
+   1. Test case: Edits a warehouse with existing WAREHOUSE_NAME in list e.g. `edit ct/w i/1 n/Alice Warehouse` followed by `edit ct/w i/2 n/Alice Warehouse`<br>
+      Expected: An error will occur and a message will be displayed, stating that a warehouse with duplicate
+      WAREHOUSE_NAME cannot be added into the list. WarehouseList on GUI remain unchanged.
+
+## Exiting CLI-nic
+
+1. Exit command format: `exit`
+
+   1. Test case: Exit command with no additional arguments e.g. `edit`<br>
+      Expected: CLI-nic closes with current state of data saved.
+   1. Test case: Exit command with additional arguments e.g. `exit test` or `exit ct/s`<br>
+      Expected: Similar to previous.
+
+## Listing CLI-nic
+
+1. List command format: `list`
+
+   1. Test case: List command with no additional arguments e.g. `list`<br>
+      Expected: CLI-nic lists all suppliers and warehouses data in CLI-nic.
+   1. Test case: List command with additional arguments e.g. `list test` or `list i/1`<br>
+      Expected: Similar to previous.   
+
+### Viewing a Supplier
+
+1. View command format: `view ct/TYPE i/INDEX`
+
+   1. Test case: View command with complete prefixes e.g. `view ct/s i/1`<br>
+      Expected: SupplierList updates to show only supplier at index 1. Products associated with the supplier and their details are shown in the command result box.
+   1. Test case: View command with missing prefixes e.g `view ct/s` or `view`<br>
+      Expected: SupplierList will not be updated to show only supplier at index 1. Error details will be shown in the response message, indicating that it is an invalid command format. A help message for view command will also be displayed to guide user accordingly. SupplierList on GUI remains unchanged.
+   1. Test case: View command with index larger than range of supplier list displayed e.g.`view ct/s i/x` (where x is larger than the displayed list size)<br>
+      Expected: Similar to previous.
+
+## Viewing a Warehouse
+
+1. View command format: `view ct/TYPE i/INDEX`
+
+   1. Test case: View command with complete prefixes e.g. `view ct/w i/2`<br>
+      Expected: WarehouseList updates to show only supplier at index 2. Products associated with the warehouse and their details are shown in the command result box.
```

Comment:
Minor typo here! "show only supplier"

Reply:
Will do the change! Thanks for spotting the error